articles/healthcare-apis/fhir/use-postman.md

```diff
@@ -72,7 +72,7 @@ Enter `{{fhirurl}}/metadata` in the `GET`request, and select `Send`. You should
 
 <a name='get-azure-ad-access-token'></a>
 
-## Get Microsoft Entra access token
+## Get Microsoft Entra access token using Client Credential grant type
```

Comment:
This step is not required to set up . I would request to state at the start of the document, there are two options to access FHIR service using postman - service principal and client credentials.

Reply:
Explicitly state this is option available if user wants to access FHIR service using user account.